### PR TITLE
Adjust banner for Dart 3 release

### LINKED ISSUE #4817: [Dart 3] Update banner
I think the banner should link to the Dart 3 migration guide #4808, maybe to individual feature pages ("...Dart 3 introduces the new features records, patterns, class modifiers...")

### PR DESCRIPTION
**Pull request by MaryaBelanger**
Staged banner

Fixes #4817

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <div class="banner">
   <p class="banner__text">
-    Dart and Flutter return to Google I/O on May 10! <a href="https://io.google/2023/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_banner&utm_content=" class="no-automatic-external">Register now</a>
+    Dart 3 is out today, along with Google I/O!
```
Comment (parlough) [inline src/_includes/banner.html:3]: Optional: Maybe increase the desire to read with the two big features?

```suggestion
    Dart 3 is here with records and patterns, along with Google I/O!
```

File: src/_includes/banner.html [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+    <br>
+    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Check out the blog post</a> and
```
Comment (parlough) [inline src/_includes/banner.html:5]: Will this link still work once published?

Comment (MaryaBelanger) [reply src/_includes/banner.html:5]: Yeah, the ID is what matters, e.g. for an already-published blog https://medium.com/p/547ef3d6459b takes you to the full `/dartlang/<title>-<id>` URL

Comment (parlough) [reply src/_includes/banner.html:5]: Cool, just wanted to make sure :)

File: src/_includes/banner.html (hunk 1 above)
Comment (MaryaBelanger) [reply src/_includes/banner.html:3]: I'm going to change the second part to "alongside Google I/O", just to not say "with" twice. Lmk if that sounds/looks weird!

Comment (parlough) [reply src/_includes/banner.html]: Sure "alongside" sounds good to me!

File: src/_includes/banner.html (hunk 2 above)
Comment (MaryaBelanger) [reply src/_includes/banner.html:5]: I just learned that today!

**Review by parlough — APPROVED** [review]
Thanks!

**Review by parlough — APPROVED** [review]
Thanks!

**Pull request merged**